components/dash-core-components/src/fragments/Dropdown.react.js

```diff
@@ -57,7 +65,7 @@ export default class Dropdown extends Component {
         } = this.props;
         const {filterOptions} = this.state;
         let selectedValue;
-        if (type(value) === 'array') {
+        if (type(value) === 'Array') {
```

Comment:
😱 wow wild that we didn't catch this before. Can we add a quick test to lock this down?